llvm/utils/git/code-format-helper.py

```diff
@@ -357,9 +376,17 @@ def hook_main():
         changed_files = args.changed_files.split(",")
 
     failed_formatters = []
+    comments = []
     for fmt in ALL_FORMATTERS:
         if not fmt.run(changed_files, args):
             failed_formatters.append(fmt.name)
+        if fmt.comment:
+            comments.append(fmt.comment)
+    
+    if len(comments):
```

Comment:
This seems like it also needs to be gated behind `args.write_comment_to_file`?

Reply:
It effectively is, since comments will never have a non-zero length when args.write_comment_to_file is False.